tensorboard/webapp/metrics/views/card_renderer/scalar_card_test.ts

```diff
@@ -2270,14 +2270,9 @@ describe('scalar card', () => {
           start: {step: 20},
           end: {step: 40},
         });
-        store.overrideSelector(getMetricsCardRangeSelectionEnabled, true);
-        store.overrideSelector(getCardStateMap, {
-          card1: {
-            dataMinMax: {
-              minStep: 0,
-              maxStep: 100,
-            },
-          },
+        store.overrideSelector(getMetricsCardTimeSelection, {
```

Comment:
This can now be much simpler because the time selection is now fully derived in the selector layer